packages/gatsby/src/utils/start-server.ts

```diff
@@ -427,20 +382,6 @@ module.exports = {
     }
   )
 
-  // Disable directory indexing i.e. serving index.html from a directory.
-  // This can lead to serving stale html files during development.
-  //
-  // We serve by default an empty index.html that sets up the dev environment.
-  app.use(developStatic(`public`, { index: false }))
-
-  const webpackDevMiddlewareInstance = webpackDevMiddleware(compiler, {
-    publicPath: devConfig.output.publicPath,
-    stats: `errors-only`,
-    serverSideRender: true,
-  })
-
-  app.use(webpackDevMiddlewareInstance)
-
```

Comment:
What's the impact of this? We will now output browser dev bundle to `fs` instead of using some form of in-memory-fs used by this middleware?

Is this actually needed for this PR?